src/components/ContextualSaveBar/tests/ContextualSaveBar.test.tsx

```diff
@@ -12,58 +12,111 @@ describe('<ContextualSaveBar />', () => {
   };
 
   it('calls the contextual save bar on mount with correct values', () => {
-    const {frame} = mountWithContext(<ContextualSaveBar {...props} />);
-    expect(frame.setContextualSaveBar).toHaveBeenCalledWith({
+    const mockFrameContext = {
+      frame: {
+        setContextualSaveBar: jest.fn(),
+        removeContextualSaveBar: jest.fn(),
+        showToast: noop,
+        hideToast: noop,
+        startLoading: noop,
+        stopLoading: noop,
+      },
+    };
```

Comment:
This should be moved into a function that generates a mock frame context and accepts spies for methods available there.